rosetta-cli-conf/testnet/lto.ros

```diff
@@ -41,21 +41,21 @@ create_account(1){
   }
 }
 
-transfer(10){
+transfer(5){
   transfer{
     transfer.network = {"network":"testnet", "blockchain":"lto"};
     currency = {"symbol":"LTO", "decimals":8};
     sender = find_balance({
       "minimum_balance":{
-        "value": "5000000000",
+        "value": "500000000",
         "currency": {{currency}}
       }
     });
 
     // Set the recipient_amount as some value <= sender.balance-max_fee
     max_fee = "100000000";
     available_amount = {{sender.balance.value}} - {{max_fee}};
-    recipient_amount = random_number({"minimum": "1", "maximum": {{available_amount}}});
+    recipient_amount = "10000000";
```

Comment:
Can we move this to fixed values for minimum and maximum? Like 

random_number({"minimum": "10000000", "maximum": "30000000"});

Reply:
done